packages/snaps-cli/src/config.ts

```diff
@@ -639,6 +663,14 @@ export const SnapsWebpackConfigStruct = object({
     false,
   ),
 
+  typescript: defaulted(
+    object({
+      enabled: defaulted(boolean(), false),
+      configFile: defaulted(string(), 'tsconfig.json'),
```

Comment:
```suggestion
      configFile: defaulted(file(), resolve(process.cwd(), 'tsconfig.json')),
```

Reply:
`file()` resolves to `process.cwd()` on it's own

Reply:
Only if a file is specified. The coercion isn't used for default values

Reply:
Maybe it does actually 😅